### PR TITLE
New doc page about circulars and updated style guide

### PR DESCRIPTION
**Pull request by jracusin**

### CODE REVIEW

**Review by dakota002 — APPROVED** [review]

File: app/routes/circulars/circulars.lib.ts [hunk 1]
```diff
@@ -76,7 +76,9 @@ export function subjectIsValid(subject: string) {
   if (subject.length)
     return (
       !emailIsAutoReply(subject) &&
-      validSubjectKeywords.some((x) => subject.startsWith(x))
+      validSubjectKeywords.some((x) =>
+        subject.toLowerCase().startsWith(x.toLowerCase())
+      )
```
Comment (lpsinger) [inline app/routes/circulars/circulars.lib.ts:81]: @dakota002, would you please pull out this change separately? It's OK to push this fix directly to main, if you like.

Comment (dakota002) [reply app/routes/circulars/circulars.lib.ts:81]: Yup, got it!

File: app/routes/circulars/circulars.lib.ts
Comment (lpsinger) [inline app/routes/circulars/circulars.lib.ts:1]: Do the changes to this list reflect the current state of GCN Classic's keyword list? If not, then we need to postpone updating this list until the migration.

File: app/routes/docs/styleguide.md [hunk 2]
```diff
@@ -112,13 +110,9 @@ Acceptable content for the GCN Circulars falls within only four areas:
...
+- Do not neglect to mention potential systematic errors if they are expected
 - Do not declare an event start time if you likely missed the beginning
 
 ## Do's
 
 - Work together to discover secrets of the Universe
-
-### Acknowledgements
-
-The GCN Circular service has benefited from numerous discussions and beta-testing with many people, most notably Roland Vanderspek, Ralph Wijers, James Rhoads, and Hye-Sook Park, plus the people at the 4th Huntsville GRB Workshop evening SIG session and the Woods Hole GRB Workshop. The overwhelming development work was performed by Scott Barthelmy. This Style Guide was adapted from the original with input from Kevin Hurley.
```
Comment (lpsinger) [inline app/routes/docs/styleguide.md:124]: Why remove the acknowledgements? It is probably true that these people established norms that have held for decades.

File: app/routes/docs/submitting.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+handle:
+  breadcrumb: Circulars Documentation - Submitting
```
Comment (lpsinger) [inline app/routes/docs/submitting.mdx:3]:
```suggestion
  breadcrumb: Circulars - Submitting
```

**Review by lpsinger — CHANGES_REQUESTED** [review]
Please move the documentation subsections down one level on the directory tree in /app/routes/docs/circulars.

File: app/routes/docs/styleguide.md (hunk 2 above)
Comment (jracusin) [reply app/routes/docs/styleguide.md:124]: The style guide isn't the right place for acknowledgements.  It should go into the About page, or history.

File: app/routes/circulars/circulars.lib.ts
Comment (lpsinger) [reply app/routes/circulars/circulars.lib.ts:1]: OK. Could we get this into a separate PR?

**Comment by jracusin** [comment]
PR #755 separates out circulars.lib.ts comments

**Comment by jracusin** [comment]
> Please move the documentation subsections down one level on the directory tree in /app/routes/docs/circulars.

done

**Review by lpsinger — CHANGES_REQUESTED** [review]
- [x] Make new content displayed conditionally on whether the `CIRCULARS` feature flag is enabled. See documentation on feature flags at https://dev.gcn.nasa.gov/docs/contributing/feature-flags (will be deployed to production shortly).
- [x] Rebase, and use the new `SideNavSub` component for the second-level nav menu so that the submenu items are only displayed when the user is in the Circulars section of the docs.

**Comment by jracusin** [comment]
I think I messed up the rebase

**Comment by lpsinger** [comment]
> I think I messed up the rebase

Yes. I see unrelated changes that were already on main.

**Comment by lpsinger** [comment]
@jracusin, in the future, I suggest using `git rebase`, not `git merge`, for this reason.

**Comment by lpsinger** [comment]
Due to the messy merges, this PR continues as #785.

**Pull request closed without merging**